bridges/GuardianBridge.php

```diff
@@ -0,0 +1,96 @@
+<?php
+class GuardianBridge extends FeedExpander {
```

Comment:
In line with the [docs](https://github.com/RSS-Bridge/rss-bridge/wiki/How-to-create-a-new-Bridge), the bridge class and filename should probably be `TheGuardianBridge`

Reply:
Fixed. And I'm sorry I didn't  know that you could directly commit reviews. I changed it manually.